scripts/install.go

```diff
@@ -0,0 +1,62 @@
+package scripts
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+//go:embed setup/setup.sh
+//go:embed setup/config-template.toml
+//go:embed setup/coriolis-snapshot-agent.service.sample
+var setupContent embed.FS
+
+const (
+	setupTmpDir = "/tmp/setup"
+)
+
+func writeEmbeddedFile(filename string, mode fs.FileMode) error {
+	fileContent, err := setupContent.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
+	return os.WriteFile(filepath.Join("/tmp", filename), fileContent, mode)
+}
+
+func RunInstall() {
+	if err := os.MkdirAll(setupTmpDir, 0700); err != nil {
```

Comment:
Use [os.MkdirTemp()](https://pkg.go.dev/os#MkdirTemp) to set up the temp dir. This ensures that there is no risk of collision with other processes or goroutines. 

```suggestion
        dir, err := os.MkdirTemp("", "snapshot-agent")
	if err != nil {
```